geoapps/drivers/base_inversion.py

```diff
@@ -341,34 +342,35 @@ def get_tile_misfits(self, tiles):
             [],
         )
         for tile_id, local_index in enumerate(tiles):
```

Comment:
Whoops, get_tile_misfits should have been replaced by MisfitFactory.  This should have been deleted

Reply:
True, and deleted